utils/test/testinput/gdas_icecabi2ioda.yaml

```diff
@@ -0,0 +1,13 @@
+provider: ABI 
+window begin: 2024-06-18T15:00:00Z
+window end: 2024-06-19T21:00:00Z
+output file: icec_abi.ioda.nc
+#ocean basin: RECCAP2_region_masks_all_v20221025.nc

```

Comment:
Is this supposed to be commented out? If so, is it worth deleting or do we want to keep it for future testing?

Reply:
It is currently not part of the ctest, and needs to be used when processing obs.